e2e/cypress/integration/enterprise/guest_accounts/guest_invitation_ui_spec.js

```diff
@@ -244,7 +246,7 @@ describe('Guest Account - Guest User Invitation Flow', () => {
         invitePeople(user1.email, 1, user1.username);
 
         // * Verify the content and message in next screen
-        verifyInvitationError(user1.username, 'This person is already a member.');
+        verifyInvitationSuccess(user1.username, 'An invitation email has been sent.');
```

Comment:
@josephbaylon This test is failing because verifyInvitationSuccess checks for successful invitations and h2.subtitle span has the text "1 person has been invited" whereas it looks like for an existing user, the invite should return "1 invitation was not sent".

Reply:
Thanks @prapti for the feedback.

@saturninoabril @srkgupta Can you guys please also run this locally and see if you're getting the same results as @prapti ? For me, the code passes fine but I may have something missing. Here are my steps,

1. Checkout this PR on mattermost-web
2. Checkout master on mattermost-server 
3. run `make nuke && make test-data && make run`
4. Run the test

Reply:
Your process seems fine @josephbaylon. It might very well be my setup that's different because I don't always run `make nuke` but I do restart my server for every PR change to make sure that changes are captured. (I meant to point it out on the original post, but forgot :) )

Reply:
Failing on me too both on existing and fresh intallation.

![Screen Shot 2020-01-31 at 9 56 16 AM](https://user-images.githubusercontent.com/5334504/73506395-6ac9d880-4410-11ea-96c7-1254a14a2102.png)

Reply:
@josephbaylon as per the correct expectation of the testcase, we should actually be receiving an error message if we are trying to invite an existing member who is already part of the team. Hence the existing assertion was correct. If the app shows a success message by any chance, it might be because of a bug in the application.

Reply:
Updated